src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,127 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_CAPACITY = 10;
+    private T[] elements;
+    private int currentCapacity;
+    private int size = 0;
+
+    public ArrayList() {
+        this.elements = (T[]) new Object[INITIAL_CAPACITY];
+    }
+
+    public void extendArray() {
```

Comment:
should this method be public?
and read this link please - https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents